### DISCUSSION TITLE
how to show dropdown with multiple example

### DISCUSSION THREAD

**Discussion by hajikkpvendor** [swaggo/swag#1356] [Q&A]
how to show dropdown with multiple example

**Comment by kevin-you2** [comment]
We will need openapi 3.0 first to proceed.

**Comment by Nerzal** [comment]
Good news: The OpenAPI V3.1.0 support is there on the V2 branch.
Support for multiple examples can now be implemented

**Comment by kevin-you2** [comment]
https://github.com/swaggo/swag/issues/1051
Will this feature be possible now?

**Reply by Nerzal** [reply]
Swaggo still only generates a single spec.
